templates/al2023/runtime/bin/limit-c-states.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -o pipefail
+set -o nounset
+set -o errexit
+
+if [ "$ACCELERATOR_VENDOR" == "neuron" ] || [ "$ACCELERATOR_VENDOR" == "nvidia" ]; then
```

Comment:
I think this will go along with [Carter's comment above](https://github.com/awslabs/amazon-eks-ami/pull/1924/files#r1727776198) for the use case where EFA is used, regardless of whether or not accelerators or GPUs are utilized/present

Reply:
sounds good